typeshed/pyodbc.pyi

```diff
@@ -0,0 +1,466 @@
+# Attention:
```

Comment:
is this generated or from where does it come and how can we update it

Reply:
It's a copy from the pyodbc repo I'll put a link in the file.